src/Microsoft.Identity.Web/WebAppAuthenticationBuilderExtensions.cs

```diff
@@ -65,10 +64,59 @@ public static partial class WebAppAuthenticationBuilderExtensions
             Action<MicrosoftIdentityOptions> configureMicrosoftIdentityOptions,
             string openIdConnectScheme = OpenIdConnectDefaults.AuthenticationScheme,
             string cookieScheme = CookieAuthenticationDefaults.AuthenticationScheme,
+            bool subscribeToOpenIdConnectMiddlewareDiagnosticsEvents = false) =>
```

Comment:
@jmprieur I added an overload without custom cookie options. Should I remove it? We can just have 1 overload with config name and 1 with config actions?

Reply:
I don't understand the question, @pmaytak

Reply:
`WebAppAuthenticationBuilderExtensions` now has three methods. The difference between 2 and 3 is that 3 has cookie options parameter. I just thought maybe we should not add 2, and just have 1 and 3?
```csharp
public static AuthenticationBuilder AddMicrosoftWebApp(
            this AuthenticationBuilder builder,
            IConfiguration configuration,
            string configSectionName = "AzureAd",
            string openIdConnectScheme = OpenIdConnectDefaults.AuthenticationScheme,
            string cookieScheme = CookieAuthenticationDefaults.AuthenticationScheme,
            bool subscribeToOpenIdConnectMiddlewareDiagnosticsEvents = false)
```
```csharp
public static AuthenticationBuilder AddMicrosoftWebApp(
            this AuthenticationBuilder builder,
            Action<OpenIdConnectOptions> configureOpenIdConnectOptions,
            Action<MicrosoftIdentityOptions> configureMicrosoftIdentityOptions,
            string openIdConnectScheme = OpenIdConnectDefaults.AuthenticationScheme,
            string cookieScheme = CookieAuthenticationDefaults.AuthenticationScheme,
            bool subscribeToOpenIdConnectMiddlewareDiagnosticsEvents = false)
```
```csharp
public static AuthenticationBuilder AddMicrosoftWebApp(
            this AuthenticationBuilder builder,
            Action<OpenIdConnectOptions> configureOpenIdConnectOptions,
            Action<MicrosoftIdentityOptions> configureMicrosoftIdentityOptions,
            Action<CookieAuthenticationOptions> configureCookieAuthenticationOptions,
            string openIdConnectScheme = OpenIdConnectDefaults.AuthenticationScheme,
            string cookieScheme = CookieAuthenticationDefaults.AuthenticationScheme,
            bool subscribeToOpenIdConnectMiddlewareDiagnosticsEvents = false)
```

Reply:
Oh, I understand. Yes, I agree. Only 1 and 3.
Thanks @pmaytak